docs/develop/contracts/rust/intro.md

```diff
@@ -356,126 +502,202 @@ assert_eq!(1, contract.get_num());
+> Please call the contract deployed to NEAR account X. On that contract there's a method called
+`increment` that takes no additional arguments. Oh, and we happen to be calling this contract 
+using keys from account X, too.
+
+Contract methods can be called from other NEAR accounts easily. Please see
+[the examples page](https://near.dev) for more information.
+
+
+#### Decrement
+
+Next, call the `decrement` method in the same way:
 
-Decrement:
 ```bash
 near call YOUR_ACCOUNT_HERE decrement --accountId YOUR_ACCOUNT_HERE
 ```
 
-Check counter:
+You should see a response on your terminal:
+
+```
+Scheduling a call: YOUR_ACCOUNT.testnet.decrement()
+Receipt: 3HiRrL4fg9Q62VV9aVfAdRk8bQ5nYEYTni9482qjPJ71
+	Log [YOUR_ACCOUNT.testnet]: Decreased number to 0
+	Log [YOUR_ACCOUNT.testnet]: Make sure you don't overflow, my friend.
+Transaction Id 7jVgp677evM7srG697bVWErErzieBWExLvkSiBfvZ8YC
+```
+
+#### Check counter value
+
+To check the current counter value, call the `get_num` method:
+
 ```bash
 near view YOUR_ACCOUNT_HERE get_num --accountId YOUR_ACCOUNT_HERE
 ```
 
-The smart contract is alive on the blockchain!
+The method should provide a response:
+
+```
+View call: YOUR_ACCOUNT.testnet.get_num()
+0
+```
+
+## Next steps
 
-This example is as bare bones as it gets, but illustrates all the moving parts associated with writing a smart contract with Rust. Admittedly, it's a poor example when it comes to creating anything user-facing.
+This example is as bare bones as it gets, but illustrates all the moving parts associated
+with writing a smart contract with Rust. Admittedly, it's a poor example when it comes to
+creating anything user-facing.
 
-Now that we're familiar with the build process, a natural next step might be to check out `create-near-app`. This project includes another Rust smart contract but has an interface. With `create-near-app` many of the steps we performed on the command line are wrapped neatly into build scripts.
+Now that you're familiar with the build process, a natural next step is to check out
+`create-near-app`. This project includes another Rust smart contract but has an interface.
+With `create-near-app` many of the steps we performed on the command line are wrapped
+neatly into build scripts.
 
-[Read more](https://github.com/near/create-near-app/) about `create-near-app` or try it out now by running:
+[Read more](https://github.com/near/create-near-app/) about `create-near-app` or try it
+out now by running:
 
 ```bash
 npx create-near-app --contract=rust new-awesome-app
 ```
 
-Follow the instructions to set up a simple Rust smart contract with a React front-end. Happy coding!
+Follow the instructions to set up a simple Rust smart contract with a React front-end.
+Happy coding!
 
-## Troubleshooting
+## Versioning for this article
 
-If `near` commands return an error containing:
-
->Cannot deserialize the contract state.
+At the time of this writing, this example works with the following versions:
```

Comment:
Love that you added the versioning with this! Nice job :)

Reply:
Thanks! 😄